src/icc-tags.js

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import {iccTags, iccProfile} from './icc-tag-names';
+import {getStringFromDataView, getUnicodeStringFromDataView} from './utils';
+
+export default {
+    read
+};
+
+const PROFILE_HEADER_LENGTH = 84;
+
+function read(dataView, iccData) {
```

Comment:
From what I understand this function isn't tested anywhere. There should probably be a `icc-test-spec.js`.

Reply:
It is tested by full ICC  profile file load. Adding additional test here would just be taking ICC file and splitting into data - basically duplicating what already happens.

Reply:
That one is only using `parseTags`, not `read`. (I'm working on adding a test coverage tool to better visualize these things.)